### ISSUE TITLE
Audio upload does not work with ModSecurity enabled

### ISSUE THREAD

**Issue by mapkyca** [idno/Known#715]
ModSecurity: Access denied with code 44 (phase 2). Match of "eq 0" against "MULTIPART_UNMATCHED_BOUNDARY" required. [file "/etc/modsecurity/modsecurity.conf"] [line "84"] [msg "Multipart parser detected a possible unmatched boundary."] [tag "I do not thing you wanted to do that..."] [hostname "mapkyca.com"] [uri "/media/edit"] [unique_id "VLT-NU4uZacAAGTiKQQAAAAF"]

**Comment by benwerd** [comment]
Well that's exceptionally weird. And annoying.

On Tue, Jan 13, 2015 at 3:21 AM, Marcus Povey <notifications@github.com>
wrote:

> ModSecurity: Access denied with code 44 (phase 2). Match of "eq 0" against
> "MULTIPART_UNMATCHED_BOUNDARY" required. [file
> "/etc/modsecurity/modsecurity.conf"] [line "84"] [msg "Multipart parser
> detected a possible unmatched boundary."] [tag "I do not thing you wanted
> to do that..."] [hostname "mapkyca.com"] [uri "/media/edit"] [unique_id
> "VLT-NU4uZacAAGTiKQQAAAAF"]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idno/idno/issues/715>.
>



-- 
Ben Werdmuller
<http://goog_1933028737>
benwerd.com | werd.io

+1 (312) 488-9373

**Comment by benwerd** [comment]
This is a guide to fixing for WordPress: http://wpsecure.net/2012/01/using-mod_security-2-with-wordpress/

It sounds like ModSecurity basically breaks everything. So, yay.

**Comment by mapkyca** [comment]
For the record, I took the brute force approach of removing the offending rule. 

This is the first time modsec has broken known, and I've never had it puke on wordpress. This is however the first time I've tried to upload audio.

It does however break all the time on mantis, where for some reason (possibly mantis' handling of POST data) it interprets a sequence of "---" as a content boundary. 

Audio should be being base64 encoded, so I'm not sure why that'd happen.
